tests/components/test_nlp.py

```diff
@@ -72,33 +72,34 @@ def test_process(kwik_e_mart_nlp):
 def test_query_info_contains_language_information(kwik_e_mart_nlp):
     file_to_test = None
     for file in kwik_e_mart_nlp.resource_loader.file_to_query_info.keys():
-        if 'train.txt' in file:
+        if "train.txt" in file:
             file_to_test = file
             break
-    query = kwik_e_mart_nlp.resource_loader.file_to_query_info[file_to_test]['queries'][0].query
-    assert query.language == 'en'
-    assert query.locale == 'en_CA'
+    query = kwik_e_mart_nlp.resource_loader.file_to_query_info[file_to_test]["queries"][
+        0
+    ].query
```

Comment:
boo. Let's assign `query_info = kwik_e_mart_nlp.resource_loader.file_to_query_info` at the top of the function, and use `query_info` here and line 74.